source/shared/cpp/AdaptiveCardsSharedModel/AdaptiveCardsSharedModelUnitTest/EverythingBagel.json

```diff
@@ -29,7 +29,7 @@
 			"isSubtle": false,
 			"maxLines": 1,
 			"size": "default",
-			"weight": "default",
+			"weight": "normAl",
```

Comment:
>normAl", [](start = 14, length = 8)

is this intended?

Reply:
yep -- just making sure we have coverage for our back-compat reverse mapping :)

---
In reply to: [231253792](https://github.com/Microsoft/AdaptiveCards/pull/2151#discussion_r231253792) [](ancestors = 231253792)